libr/core/graph.c

```diff
@@ -2218,6 +2218,28 @@ R_API RAGraph *r_agraph_new(RConsCanvas *can) {
 	return g;
 }
 
+static void visual_offset (RCore *core) {
+	char buf[256];
+	r_line_set_prompt ("[offset]> ");
+	strcpy (buf, "s ");
+	if (r_cons_fgets (buf+2, sizeof (buf)-3, 0, NULL) >0) {
+		if (buf[2]=='.')buf[1]='.';
+		r_core_cmd0 (core, buf);
+	}
+}
+
+static void print_agraph (RAGraph *g, RCore *core, struct agraph_refresh_data *grd){
+	RAnalFunction **fcn = grd->fcn;
+	RAnalFunction *f = r_anal_get_fcn_in (core->anal, core->offset, 0);
+	if (f && f != *fcn){
+		*fcn = f;
+		g->need_reload_nodes = R_TRUE;
+	}
+	agraph_print (g, grd->fs, core, *fcn);
```

Comment:
I didn't want to touch that function to don't break anything xD.

I will change now and submit again